react/features/base/app/components/BaseApp.js

```diff
@@ -125,6 +126,7 @@ export default class BaseApp extends Component<*, State> {
                     <Provider store = { store }>
                         <Fragment>
                             { this._createMainElement(component) }
+                            <DialogContainer />
```

Comment:
I'm not familiar with BaseApp and how it related to AbstractApp and how that's somehow extended or used by App. Does adding DialogContainer at this level mess with web's DialogContainer? I don't see this DialogContainer wrapped in AtlasKitThemeProvider. If this DialogContainer also gets used on welcomepage for web then maybe web's welcome page needs it DialogContainer removed?